plugins/FelixCardReader.cpp

```diff
@@ -92,17 +93,25 @@ FelixCardReader::init(const data_t& args)
       } catch (const std::exception& ex) {
         ers::fatal(InitializationError(ERS_HERE, "Link ID could not be parsed on queue instance name! "));
       }
-      auto link_offset = 0;
-      if (linkid >= 5) { // RS FIXME: super ugly... queue names should contain tag for exact elink.
-        link_offset = 5;
-      }
-      auto tag = (linkid - link_offset) * m_elink_multiplier;
-      TLOG_DEBUG(TLVL_WORK_STEPS) << "Creating ElinkModel for target queue: " << target << " elink tag: " << tag;
-      m_elinks[tag] = createElinkModel(qi.inst);
-      if (m_elinks[tag] == nullptr) {
+      //auto link_offset = 0;
```

Comment:
is there a reason to leave this code commented, rather than remove it ?

Reply:
yes we can remove the comments

Reply:
done